portal/assets/css/style.css

```diff
@@ -7,12 +7,7 @@
  * @copyright Copyright (c) 2016-2017 Jerry Padgett <sjpadgett@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
-
-/*@import url(//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,300italic,400italic,600italic);
-@import url(//fonts.googleapis.com/css?family=Kaushan+Script);*/
-/*!
-
-!*/
+@import './../../../interface/themes/oe-common/oe-sidebar.scss';
```

Comment:
This import seems odd to bring in a scss. However, I can see the issue if you are not bringing in one of the standard themes. Is that the case?
(if so, then sounds like we get to build a portal css during build :) )

Reply:
btw, if we do need to work on making a portal css build, then I'll separate out and package up the build updates from my other PR and bring them in here.

Reply:
portal doesn't use an openemr theme thus is why we put in the no_main-theme in header class.
after I finish Home page sidebar and navbar rework then will likely go with openemr theming.